module1/build tools/multi-project/src/main/java/Main.java

```diff
@@ -0,0 +1,6 @@
+
+public class Main {
```

Comment:
I don't see multiple modules inside multi-project. According to the task, you need to create **one** multi-project with **two** sub-projects. Now I can see 3 projects in diffrent directories

Reply:
rebuild project, add two modules inside mulri-project